SPECS/privacyidea.spec

```diff
@@ -40,9 +41,9 @@ rm -fr /opt/privacyidea
 virtualenv /opt/privacyidea
 source /opt/privacyidea/bin/activate
 pip install --upgrade pip
+pip install -r https://raw.githubusercontent.com/privacyidea/privacyidea/v%{version}/requirements.txt
```

Comment:
There could be corner cases where the git repo contains another requirement in the tags than the python package index.
Should we install the privacyidea from github completely? We do this with the ubuntu packages.
I am not sure due to which problems we decided to install from pypi...

Reply:
I'll test installing it from github.

Reply:
fixed in ffdb7b0